### DISCUSSION TITLE
For bizarre reasons, I am needing to serve http websites, but Vercel does not allow this

### DISCUSSION THREAD

**Discussion by Johnrobmiller** [vercel/next.js#25130] [Ideas]
### Describe the feature you'd like to request

My feature request is to allow Next.js to allow the serving of http (as opposed to https) sites.  Why I need this request is a bit of a story.  Very soon, my boss will be presenting a published demo of a site I'm working on to some investors.  The back-end API guy doesn't have his stuff https certified yet.  As a result, I can't make any http requests on my deployed Vercel site because of the https/http discrepency (i.e., that type of action is outlawed in most browsers).  

Now, as I was typing this, I heard back from my boss, who said that the API will be https before the big meeting, but still...

### Describe the solution you'd like

(should be obvious based on context)

### Describe alternatives you've considered

Publishing my Next.js site on a non-Vercel host that allows temporarily using http instead of https.

**Comment by wwwdepot** [comment]
You can use an API route to proxy all request to your non-https backend

**Comment by Johnrobmiller** [comment]
Good idea.  Thankfully I don't have to configure any of that anymore, but that's definitely a viable solution.
